install.hs

```diff
@@ -4,23 +4,39 @@
   --resolver nightly-2018-12-15
```

Comment:
Any decision on whether to use `shake.yaml` instead of `stack script`?

Reply:
Will change back